docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.8'
```

Comment:
Compose is not needed at all, Kubernetes will be orchestrator.

Reply:
Removed `docker-compose.yaml`